build-tools/automation/yaml-templates/commercial-build.yaml

```diff
@@ -19,14 +19,14 @@ steps:
   inputs:
     forceReinstallCredentialProvider: true
 
-- task: provisionator@2
-  displayName: Install Xcode
-  inputs:
-    github_token: $(GitHub.Token)
-    provisioning_script: ${{ parameters.xaSourcePath }}/build-tools/provisioning/xcode.csx
-    provisioning_extra_args: '-v -v -v -v'
-  env:
-    PROVISIONATOR_CHANNEL: ${{ parameters.provisionatorChannel }}
+# - task: provisionator@2
+#   displayName: Install Xcode
+#   inputs:
+#     github_token: $(GitHub.Token)
+#     provisioning_script: ${{ parameters.xaSourcePath }}/build-tools/provisioning/xcode.csx
+#     provisioning_extra_args: '-v -v -v -v'
+#   env:
+#     PROVISIONATOR_CHANNEL: ${{ parameters.provisionatorChannel }}
```

Comment:
We should be able to continue to use provisionator, though I am still trying to work through issues related to monodroid. I think we can revert the provisionator related changes at least though.